agent/php_user_instrument.c

```diff
@@ -644,6 +552,11 @@ void nr_php_user_function_add_declared_callback(const char* namestr,
   }
 }
 
+#if ZEND_MODULE_API_NO >= ZEND_7_4_X_API_NO
```

Comment:
Seems like this >= 7.4 block might flow better with the code if it was in the block up at line 79 above?

Reply:
I think 32a63ed07d08f5af1784a69b42aec22ea398adc3 addressed this remark.